reflectance_api.py

```diff
@@ -32,7 +32,7 @@ def getReflectanceListbyPoint1992(point):
         'INFO_FORMAT': 'text/html'
     }
 
-    resp = service_api.getRequest(params=PARAMS, url=URL)
+    resp = service_api.getRequest(params=PARAMS, url=ODBICIOWOSC_WMS_URL)
```

Comment:
O dobrze że właśnie nazwałaś ten url w taki jednoznaczny sposób. Widziałem że niewszędzie tak było. Proszę popraw to dla innych produktów